### DISCUSSION TITLE
Rotate MarkerView relative to map, not screen

### DISCUSSION THREAD

**Discussion by msi89** [rnmapbox/maps#2822] [Q&A]
Normally it looks that:

<img width="324" alt="Screenshot 2023-04-27 at 23 58 37" src="https://user-images.githubusercontent.com/20075620/234991608-51fa9197-184b-4401-aa32-b2845d2df8d1.png">

But when i rotate the map, looks that:
<img width="268" alt="Screenshot 2023-04-27 at 23 59 03" src="https://user-images.githubusercontent.com/20075620/234991957-eb47829d-e6e3-4dae-9ea2-2d7cae437ae0.png">

In android SDK, we can do that  :
```kotlin
val pointAnnotationManager: PointAnnotationManager = binding.mapView.annotations.createPointAnnotationManager(AnnotationConfig(layerId = ""))
pointAnnotationManager.iconRotationAlignment = IconRotationAlignment.MAP

 val pointAnnotationOptions : PointAnnotationOptions = PointAnnotationOptions()
                .withPoint(marker.position!!)
                .withIconRotate(marker.rotation)
                .withIconImage(icon!!)
                .withIconSize(1.0)
pointAnnotationManager.create(pointAnnotationOptions)
```

how can do that with MarkerView?
```jsx
<MarkerView id="marker" coordinate={[4.0, -4.0]}>
          <View style={styles.center}>
      </View>
</MarkerView>

**Comment by mfazekas** [comment]
@msi89 use PointAnnotation, 

MarkerView is implemented using [ViewAnnotation](https://docs.mapbox.com/android/maps/api/10.12.2/mapbox-maps-android/com.mapbox.maps.viewannotation/-view-annotation-manager/add-view-annotation.html)

**Reply by msi89** [reply]
[mfazekas](https://github.com/mfazekas), Thanks for the answer but even PointAnnotation does not solve the problem. It is a component that follows a route like in the example:   https://docs.mapbox.com/mapbox-gl-js/example/animate-point-along-route/
